test/MockSmartStore.js

```diff
@@ -32,44 +32,44 @@
  */
 
 var MockSmartStore = (function(window) {
-    // Private
```

Comment:
Making those members of the MockSmartStore class (otherwise they would be shared between instances of that class).
Also, it makes mock smartstore a lot easier to debug with the dev tools.